source/kubernetes/direct-csi.rst

```diff
@@ -0,0 +1,135 @@
+==============================================
+MinIO Direct Container Storage Interface (CSI)
+==============================================
+
+.. default-domain:: minio
+
+.. contents:: On This Page
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+The MinIO Direct Container Storage Interface (CSI) driver for Kubernetes
+supports creating Persistant Volumes (``PV``) from disks attached to a 
+:kube-docs:`Node <concepts/architecture/nodes/>`. The
```

Comment:
One addition (feel free to rephrase): Direct CSI is typically used, though not limited to,  deployments using Minio on k8s running on top of bare-metal servers with JBOD. 
The main goal is to have customers land here when searching for those terms.

Reply:
Done